files/es/learn/css/css_layout/floats/index.md

```diff
@@ -328,7 +314,7 @@ body {
 }
 ```
 
-{{ EmbedLiveSample('Float_3', '100%', 600) }}
+{{ EmbedLiveSample('Limpiando floats', '100%', 600) }}
```

Comment:
floats is English, so, I think there should be flotadores in plural
```suggestion
{{ EmbedLiveSample('Limpiando flotadores', '100%', 600) }}
```

Reply:
We should use the [leading heading/ID of the group of codes](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Live_samples#grouping_code_blocks). For this part, we should refer [this heading](https://github.com/mdn/translated-content/pull/17617/files#diff-6e214aac65d71ec2a87b5e178d4e33d1065221a2eed56a84cd3e99a3b6e14584R247). So I used "Limpiando floats". If you would like to fix this, please also correct the content of the above heading :)

Reply:
Thanks for clarifying it